src/ui/components/headers/PageHeaderWithoutPlaceholder.tsx

```diff
@@ -29,6 +29,7 @@ export const useGetHeaderHeight = () => {
   return HEADER_HEIGHT + top + 1
 }
 
+// naming : we need an empty view below header as the header is in absolute
 export const PageHeaderWithoutPlaceholder: FunctionComponent<Props> = ({
```

Comment:
Je trouve pas hyper clair l'explication je fais une proposition mais je te laisse choisir ou compléter :) 
Component naming : this component needs to be used with a PlaceHolder component that has the height of the header as it is an absolute view

Reply:
Yes je prends la proposition